### PR TITLE
fixed header for popup windows with scrollable content inside

### LINKED ISSUE #1350: Navigate window does not have fixed header
It's probably all of the windows. When the content is longer then the max-height of the window the header does not stay at the top and scrolls away as the user scrolls down.

screenshot from 2017-01-27 12-21-36
screenshot from 2017-01-27 12-21-52

### PR DESCRIPTION
**Pull request by tulinkry**
fixes #1350

Navigate window now has fixed header and scrollable content. Other windows don't need it.

screenshot from 2017-07-24 16-05-50

### CODE REVIEW

File: web/default/style.css [hunk 1]
```diff
@@ -819,10 +809,34 @@ a.scope            { /* scope              */ color: steelblue; font-weight: bol
     position: fixed;
     font-size: 12px;
     font-family: monospace;
-    overflow: auto;
+    overflow: hidden;
```
Comment (vladak) [inline web/default/style.css]: if the content is clipped, what mechanism will add the scrollbar now ?

Comment (tulinkry) [reply web/default/style.css]: The content is in the body and that is set to overflow auto

**Comment by tulinkry** [comment]
conflicts resolved

**Comment by tarzanek** [comment]
@tulinkry one more please, will merge after you resolve it

**Comment by tarzanek** [comment]
@tulinkry :) one more conflict resolve, please (and ping me on IM so I will merge asap)

**Pull request merged**